### ISSUE TITLE
Returns strings by default from styles

### ISSUE THREAD

**Issue by Olical** [Olical/react-faux-dom#68]
Right now styles default to undefined. Defaulting to strings should fix a few things mentioned at the bottom of #45. Any input on this would be cool.

**Comment by dchambers** [comment]
Hi @Olical, I just changed this line within `Element.js`:

~~~js
return props.style[styleCamelCase(name)]
~~~

to this:

~~~js
return props.style[styleCamelCase(name)] || ''
~~~

but I don't know if it needs to be more subtle than that?

**Comment by Olical** [comment]
Pretty much all I was going to do + tests :)

**Comment by dchambers** [comment]
I see, makes sense :smile:

**Comment by dchambers** [comment]
Okay, I've created a [pull-request for this](https://github.com/Olical/react-faux-dom/pull/69) in case you haven't started it already.

**Comment by Olical** [comment]
Thanks @dchambers, I'll get this released ASAP.

**Comment by dchambers** [comment]
Thanks @Olical, much appreciated 👍

**Comment by Olical** [comment]
Released in `v3.0.0`. Thanks for the report and fix.

**Comment by dchambers** [comment]
That's working for me. Thanks again @Olical!
